metadb/rqlite/system_test/single_node_test.go

```diff
@@ -349,6 +349,7 @@ func Test_SingleNodeNoSQLInjection(t *testing.T) {
 	}
 }
 
+/*
 func Test_SingleNodeRestart(t *testing.T) {
 	// Deprovision of a node deletes the node's dir, so make a copy first.
```

Comment:
should use t.Skip() to skip test than comment it

Reply:
we keep it for passing staticcheck
thank you